ote/test/clj/ote/db/service_generators.clj

```diff
@@ -6,12 +6,27 @@
             [clojure.spec.gen.alpha :as sgen]
             [clojure.spec.test.alpha :as stest]
             [ote.db.transport-operator :as t-operator]
+            [ote.db.transit :as transit]
             [ote.db.transport-service :as t-service]
             [ote.db.common :as common]
             [ote.time :as time]
             [clojure.string :as str]
             [ote.db.generators :as generators]))
 
+(def gen-effective-dates
+  (gen/hash-map
+    ::transit/effective-date generators/gen-effective-date
+    ::transit/effective-date-description generators/gen-text))
```

Comment:
Why not use the naughty string generator instead of creating a new text generator?

Reply:
Yep. good idea